x-pack/plugins/profiling/public/views/no_data_view/index.tsx

```diff
@@ -21,14 +23,30 @@ import {
   EuiTabs,
   EuiText,
 } from '@elastic/eui';
-import { i18n } from '@kbn/i18n';
 import { FormattedMessage } from '@kbn/i18n-react';
-import React, { useState } from 'react';
+import { useProfilingParams } from '../../hooks/use_profiling_params';
+import { useProfilingRouter } from '../../hooks/use_profiling_router';
+import { useProfilingRoutePath } from '../../hooks/use_profiling_route_path';
 import { AsyncStatus, useAsync } from '../../hooks/use_async';
 import { useProfilingDependencies } from '../../components/contexts/profiling_dependencies/use_profiling_dependencies';
 import { ProfilingAppPageTemplate } from '../../components/profiling_app_page_template';
 
-export function NoDataView({ subTitle }: { subTitle: string }) {
+export enum NoDataTabs {
+  Kubernete = 'kubernetes',
```

Comment:
Minor issue with misspelled enumeration
```suggestion
  Kubernetes = 'kubernetes',
```